cmake/option/option_noarch-tizen.cmake

```diff
@@ -22,7 +22,7 @@ SET(exec_prefix "${prefix}/bin")
 SET(includedir "${prefix}/include/${PROJECT_NAME}")
 SET(libdir "${prefix}/lib")
 SET(libname "${PROJECT_NAME}")
-CONFIGURE_FILE(${libname}.pc.in ${libname}.pc @ONLY)
+CONFIGURE_FILE(config/${libname}.pc.in ${libname}.pc @ONLY)
```

Comment:
Does the third `cp` in the `install` step of the spec file work? It seems to be looking for the pc file in a cmake subdir. Does that exist?

Reply:
yes that's true , I've simplified tizen spec file to rely only on cmake install